inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/task/TaskManager.java

```diff
@@ -61,13 +61,13 @@ public class TaskManager extends AbstractDaemon {
     private static final int ACTION_QUEUE_CAPACITY = 1000;
     private long lastPrintTime = 0;
     // task basic store
-    private final Store taskBasicStore;
+    private static Store taskBasicStore;
```

Comment:
Since TaskManager is not a singleton, the static taskStore might be recreated multiple times.
```
    public TaskManager() {
        taskBasicStore = initStore(
                agentConf.get(AgentConstants.AGENT_ROCKS_DB_PATH, AgentConstants.AGENT_STORE_PATH_TASK));
        taskStore = new TaskStore(taskBasicStore);
```

Reply:
> Since TaskManager is not a singleton, the static taskStore might be recreated multiple times.
> 
> ```
>     public TaskManager() {
>         taskBasicStore = initStore(
>                 agentConf.get(AgentConstants.AGENT_ROCKS_DB_PATH, AgentConstants.AGENT_STORE_PATH_TASK));
>         taskStore = new TaskStore(taskBasicStore);
> ```

Only in AgentManager will a TaskManager object be created
    public AgentManager() {
        conf = AgentConfiguration.getAgentConf();
        agentConfMonitor = Executors.newSingleThreadExecutor();
        taskManager = new TaskManager();